app/src/main/java/org/pesmypetcare/mypetcare/features/pets/PetHealthInfo.java

```diff
@@ -11,22 +12,24 @@
 
 public class PetHealthInfo {
     private Map<DateTime, Double> weight;
-    private Map<DateTime, Double> recommendedDailyKiloCalories;
+    private Map<DateTime, Double> dailyKiloCalories;
     private Map<DateTime, Integer> exerciseFrequency;
     private Map<DateTime, Integer> weeklyExercise;
     private Map<DateTime, Double> weeklyKiloCaloriesAverage;
     private Map<DateTime, Integer> washFrequency;
+    private Map<DateTime, Integer> weeklyMeals;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'weeklyMeals' could be replaced by a local variable.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5412347)